### PR TITLE
[release-4.8] Advance 4.7 bundle z-stream version

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml [hunk 1]
```diff
@@ -161,8 +172,6 @@ spec:
           strategy: {}
           template:
             metadata:
-              annotations:
-                target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```
Comment (fromanirh) [inline deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml:165]: we probably don't want to lose this

**Review by fromanirh — COMMENTED** [review]
besides the fact it seems we're losing an important annotation (please check inside), looks good

File: deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml (hunk 1 above)
Comment (cynepco3hahue) [reply deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml:165]: it 4.7, and the feature is available only for 4.8, so it should be ok. I think it was added in first place by mistake :)

Comment (fromanirh) [reply deploy/olm-catalog/performance-addon-operator/4.7.4/performance-addon-operator.v4.7.4.clusterserviceversion.yaml:165]: fair enough

**Comment by yanirq** [comment]
/hold
is this the right release ?  this fix was made only for 4.6 and 4.7  , I see you are merging it to 4.8

**Comment by cynepco3hahue** [comment]
> is this the right release ? this fix was made only for 4.6 and 4.7 , I see you are merging it to 4.8

The problem that 4.8 uses 4.7 bundle for the upgrade test.

**Comment by yanirq** [comment]
> > is this the right release ? this fix was made only for 4.6 and 4.7 , I see you are merging it to 4.8
> 
> The problem that 4.8 uses 4.7 bundle for the upgrade test.

Ack
/hold cancel
/lgtm

**Pull request merged**